lib/importproject.cpp

```diff
@@ -254,7 +256,7 @@ void ImportProject::importSln(std::istream &istr, const std::string &path)
         if (pos == std::string::npos)
             continue;
         const std::string vcxproj(line.substr(pos1+1, pos-pos1+7));
-        importVcxproj(path + Path::fromNativeSeparators(vcxproj), variables, emptyString);
```

Comment:
you are right, this is the less intrusive and more elegant version and i verified it works updated the code accordingly